src/lib.rs

```diff
@@ -732,6 +736,13 @@ impl Queue {
         }
     }
 
+    /// Set whether the recv function blocks or not
+    pub fn set_non_blocking(&mut self, non_blocking: bool) {
```

Comment:
Please name it `set_nonblocking` to be consistent with std's naming convention.

Reply:
Done.